Readme.md

```diff
@@ -91,10 +96,16 @@ $ ./y.sh test --release
 
 ## Usage
 
-`$CG_GCCJIT_DIR` is the directory you cloned this repo into in the following instructions:
+You have to run these commands, in the corresponding order:
 
 ```bash
-export CG_GCCJIT_DIR=[the full path to rustc_codegen_gcc]
+$ ./y.sh prepare
+$ ./y.sh build
```

Comment:
Oh, I missed that. There was an update in my fork that I just synced and this now needs to be:
```suggestion
$ ./y.sh build --sysroot
```